repository/t.json

```diff
@@ -2778,6 +2778,18 @@
 				}
 			]
 		},
+		{
+			"name": "Toit Syntax Highlighting",
```

Comment:
Syntax packages do more than simply highlighting, they enable various other features as well. Hence, language support packages should simply be named after the language. E.g. “css” and not “css syntax” or “css highlighting “.